src/System.Net.Security/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -25,7 +25,6 @@ public void EventSource_ExistsWithCorrectId()
         }
 
         [Fact]
-        [ActiveIssue(19909, TargetFrameworkMonikers.Uap)]
```

Comment:
You need to leave the ActiveIssue but you can set it to TargetFramework.UapAot.  NetEventSource still doesn't work in UapAot (#20470.)